Content.Server/Electrocution/Components/ElectrifiedComponent.cs

```diff
@@ -65,5 +66,20 @@ public sealed class ElectrifiedComponent : Component
 
         [DataField("siemensCoefficient")]
         public float SiemensCoefficient { get; } = 1f;
+
+        [DataField("shockNoises")]
+        public List<SoundSpecifier> ShockNoises { get; } = new()
+        {
+            new SoundPathSpecifier("/Audio/Effects/sparks1.ogg"),
+            new SoundPathSpecifier("/Audio/Effects/sparks2.ogg"),
+            new SoundPathSpecifier("/Audio/Effects/sparks3.ogg"),
+            new SoundPathSpecifier("/Audio/Effects/sparks4.ogg")
+        };
```

Comment:
this is what `SoundCollection` is for mate

Reply:
to be more clear
```suggestion
        [DataField("shockNoises")]
        public SoundSpecifier ShockNoise { get; } = new SoundCollectionSpecifier("whatever collection ID");
```

Reply:
whoops